cloudferry_devlab/cloudferry_devlab/tests/config.py

```diff
@@ -54,6 +54,9 @@
 filters_file_naming_template = 'filter_{tenant_name}.yaml'
 """Naming template for filter files."""
 
+all_tenants_filter_filename = 'all_tenants_filter.yaml'
```

Comment:
Split the update_filtering_file method to take care of generating yaml files per tenant and for all tenants.